components/input/input.html

```diff
@@ -0,0 +1,203 @@
+                    <a href="https://www.linkedin.com/in/praveen-kumar-d-06b545168/"><i class="fa fa-linkedin fa-2x"
+                            aria-hidden="true"></i></a>|
+                </div>
+                <div class="mobile-menu flex-evenly align-center">
+                    <button class="btn hamburger mobile-item bg-primary white-color"><i
+                            class="fa fa-bars fa-2x h-100"></i></button>
+                    <button class="btn hamburger-close mobile-item bg-primary white-color"><i
+                            class="fa fa-close fa-2x h-100"></i></button>
+                </div>
+            </ul>
+        </nav>
+        <div class="sidebar">
+            <ul class="sidebar-items flex-vertical">
+                <a href="/components/avatar/avatar.html">
+                    <li>Avatar</li>
+                </a>
+                <a href="/components/alerts/alerts.html">
+                    <li>Alerts</li>
+                </a>
+                <a href="/components/badges/badges.html">
+                    <li>Badges</li>
+                </a>
+                <a href="/components/buttons/buttons.html">
+                    <li>Buttons</li>
+                </a>
+                <a href="/components/cards/cards.html">
+                    <li>Cards</li>
+                </a>
+                <a href="/components/responsive-images/responsive-images.html">
+                    <li>Responsive Images</li>
+                </a>
+                <a class="active-sidebar-item" href="/components/input/input.html">
+                    <li>Input</li>
+                </a>
+            </ul>
+        </div>
+        <main class="main-section mg-md">
+            <h1>Input</h1>
+            <h3 class="mg-t-xsm">
+                Text Input
+            </h3>
+            <div class="flex-evenly mg-lg">
+                <div>
+                    <h4 class="regular-dark black-color mg-b-xxsm">Input Label</h4>
+                    <input class="input br-s" type="text" placeholder="Placeholder">
+                </div>
+            </div>
+            <div class="flex-evenly mg-t-lg">
+                <iframe src="https://carbon.now.sh/embed/myDE6WqG5jcs2RaH2PJJ"
+                    style="width: 100%; height: 192px; border:0; transform: scale(1); overflow:hidden;"
+                    sandbox="allow-scripts allow-same-origin">
+                </iframe>
+            </div>
+
+            <h3 class="mg-t-xsm">
+                Text Input Focus Variants
+            </h3>
+            <div class="flex-evenly mg-lg">
+                <div class="flex-vertical row-gap-1">
+                    <!-- Regular Input on focus -->
+                    <div>
+                        <h4 class="regular-dark black-color mg-b-xxsm">Regular</h4>
+                        <input class="input br-s" type="text" placeholder="Placeholder">
```

Comment:
Try to use class names that will reflect some meaning in place of `br-s`

Reply:
br-s is border-radius small, any name that you can suggest for it ? I wanted to just use the short for m of it, any suggestions which can make it better ?